src/lib/store/store.js

```diff
@@ -2,5 +2,6 @@ import { writable } from 'svelte/store';
 
 export const authStore = writable({
 	user: null,
-	loading: true
+	loading: true,
+	tricks: []
```

Comment:
I think we should either change the store name or create separate stores. wdyt?

Reply:
Good point! let's create a ticket for rethinking & improving the store structure